gcp_variant_transforms/testing/integration/vcf_to_bq_tests/presubmit_tests/small_tests/valid_4_0.json

```diff
@@ -2,7 +2,7 @@
   "test_name": "valid-4-0",
   "table_name": "valid_4_0",
   "input_pattern": "gs://gcp-variant-transforms-testfiles/small_tests/valid-4.0.vcf",
-  "runner": "DataflowRunner",
```

Comment:
Please make this change in another PR (specific to issue #255).

Reply:
Sorry, I meant leaving the small tests (unrelated to preprocess pipeline) as DataflowRunner and changing small_tests in another PR since it's unrelated to this change. You can use DirectRunner for the tests added here like you had before :)

Reply:
Oh, sorry about the misunderstanding. Done now.